opencraft/settings.py

```diff
@@ -677,6 +677,7 @@
 AWS_SECRET_ACCESS_KEY = env('AWS_SECRET_ACCESS_KEY', default=None)
 AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID', default=None)
 AWS_S3_BUCKET_PREFIX = env('S3_BUCKET_PREFIX', default='ocim')
+AWS_S3_DEFAULT_REGION = env('AWS_S3_DEFAULT_REGION', default='')
```

Comment:
@lgp171188 You can set the default to `us-east-1` since that [seems to be the default region](https://tasks.opencraft.com/browse/OC-4962?focusedCommentId=84576&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-84576).

Reply:
@itsjeyd, wouldn't it be better to leave the default here to `''` which will use the default region of the account and use `us-east-1` as the value for `AWS_S3_DEFAULT_REGION` in the `.env` file?

Reply:
@lgp171188 Ah, yes -- that's a better pattern 👍